src/app/screens/Onboard/ChooseConnector/index.tsx

```diff
@@ -32,6 +33,11 @@ export default function ChooseConnector() {
       description: "Connect to your LNbits account",
       logo: lnbits,
     },
+    {
+      to: "galoy",
+      title: "Galoy",
+      description: "Create or connect to a Galoy account",
```

Comment:
we need a logo here now.

Reply:
Added here: https://github.com/getAlby/lightning-browser-extension/pull/483/commits/00194782c331c397267311a3a774ba78185a1a9e